### PR TITLE
Compatibility with Liferay 7.1

### PR DESCRIPTION
**Pull request by chripalombella**
Added new Structure Field "ddm-geolocation" for Liferay 7.1
Fix mapping List object when there is only one element in the Journal Article

### CODE REVIEW

File: src/main/java/com/github/davidepastore/liferay/converter/ConvertibleJournalArticle.java [hunk 1]
```diff
@@ -186,11 +190,34 @@ protected Object getObjectValue(Element element, Object object, String title) th
...
+		} else if(type.equals("ddm-geolocation")) {
+			value = stringValue;
 		}
-		
+
 		return value;
 	}
-	
+
+		/**
```
Comment (DavidePastore) [inline src/main/java/com/github/davidepastore/liferay/converter/ConvertibleJournalArticle.java]: Can you improve the format of this line?

File: src/main/java/com/github/davidepastore/liferay/converter/ConvertibleJournalArticle.java [hunk 2]
```diff
@@ -257,10 +284,35 @@ protected Object getObjectValue(Element element, Object object, String title) th
...
 			if(!names.contains(name)){
-				names.add(name);				
+				names.add(name);
 			}
 		}
 		return names;
 	}
-	
+
+		/**
```
Comment (DavidePastore) [inline src/main/java/com/github/davidepastore/liferay/converter/ConvertibleJournalArticle.java]: Can you improve the format of this line?

**Comment by DavidePastore** [comment]
Thanks @chripalombella for your changes! 😄

**Comment by DavidePastore** [comment]
@chripalombella Just an additional question. Was the structure field "ddm-geolocation" added in the 7.1 version or it was present before?

**Comment by chripalombella** [comment]
@DavidePastore Hi, yes! The "ddm-geolocation" field was added in liferay 7.1.

**Comment by DavidePastore** [comment]
@chripalombella Version 7.1.0 published! 🎉🎉🎉

**Comment by chripalombella** [comment]
Thank you!!!

**Pull request merged**